vehicles-utils/upload-insurances/upload-insurances.py

```
@@ -57,47 +220,140 @@ def process_excel_files():
-            total_calculated *= (1 + mapped_data["insuranceTax"] / 100)
+            total_calculated *= 1 + mapped_data["insuranceTax"] / 100
         elif mapped_data["insuranceTaxType"] == "CURRENCY":
             total_calculated += mapped_data["insuranceTax"]
 
         if round(total_calculated, 4) != round(mapped_data["insuranceTotalAmount"], 4):
             raise ValueError("Diferencia en el cálculo del total de prima")
 
-        return mapped_data
+        return vehicle["id"], mapped_data
     except Exception as e:
         raise ValueError(f"Error al mapear fila: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)